packages/office-ui-fabric-react/src/components/pickers/Suggestions/Suggestions.tsx

```diff
@@ -155,6 +192,110 @@ export class Suggestions<T> extends BaseComponent<ISuggestionsProps<T>, {}> {
     );
   }
 
+  @autobind
+  public tryHandleKeyDown(keyCode: number, currentSuggestionIndex: number): boolean {
```

Comment:
Add a comment that return true means it was handled, and false means it wasn't

Reply:
This entire function seems like it should be in the controller.

Reply:
How do I move this into the controller? The suggestionsController doesn't seem to be aware of the other elements in the suggestion element? (I may be missing something)

Reply:
It's not really supposed to be aware. That said it makes sense that it would need to be aware in order to manage stuff like this. Perhaps the answer is to make SuggestionsController a react element whose render function only returns `<Suggestions>.` That way it can have a ref to suggestions, which in turn exposes an api for stuff like clicking the "search more" button. The main goal, in my mind, is to keep the "state manager" (SuggestionsController, maybe it should get renamed?)  separate from the "view" (Suggestions). That way it's easy to mix and match the two and each one is clean, there isn't a conflating of state and form.

Reply:
I've opened #3969 to address this